pbot.js

```diff
@@ -170,7 +186,7 @@ client.on('message', (message) =>
             const emoji = getEmoji(audioCommand.emoji);
             message.react(emoji);
 
-            playAudioRaw(channel, audioCommand.path);
+            isMongoCommand ? playAudioFromMongo(channel, audioCommand.path) : playAudioRaw(channel, audioCommand.path);
```

Comment:
NP: `playAudioFromMongo` takes a mongo file ID as its second argument, not technically a path:
https://github.com/jmcountryman/pbot/blob/e790e8655323463907ebf28219f52d8a3c3da426/pbot.js#L79
https://github.com/jmcountryman/pbot/blob/e790e8655323463907ebf28219f52d8a3c3da426/mongo_fs.js#L36-L44

Reply:
Updated